Assignment-2/KaprekarConstant.py

```diff
@@ -0,0 +1,25 @@
+def takeInputFromUser():
+    number = input("Enter a Number : ")
+
+    if len(number) != 4:
+        print("Invalid Input")
+        return -1
+    
+    return int(number)
+
+class KaprekarConstant:
```

Comment:
The class name might be misleading since the class seems to be handling the Kaprekar routine rather than representing a constant. A better name might be KaprekarNumber or something similar.